openweather/src/main/java/pages/HomePage.java

```diff
@@ -38,8 +45,33 @@ public void submitSearch() {
         searchCityTxt.sendKeys(Keys.ENTER);
     }
 
+    public HomePage selectMenu(String menu) {
+        new WebDriverWait(driver, TIMEOUT_IN_SECONDS)
+                .until(ExpectedConditions.elementToBeClickable
+                        (By.xpath("//a[contains(text(),'" + menu + "')]")));
```

Comment:
don't use selectors in page action method. Selector belongs to object repository which is used for mapping sector and web element id, try to separate them.